tasks/makeSearchIndex.js

```diff
@@ -24,9 +24,14 @@ if (require.main !== module) {
       id: info.id,
       summary: info.summary,
       bom: info.bom.lines
-        .map(l => l.description)
+        .map(l => {
+          const description = l.description
+          const MPN = l.partNumbers.map(p => p.part)
+
+          return [description, MPN].filter(field => field !== '').toString()
```

Comment:
Do we actually need the `.toString` and all the empty string checks?

Reply:
For one of the dev boards, the `bom` field was ", , , , , , , , , ". It will not affect the search result, yet it will unnecessary clutter to the `search_index.json`.